### DISCUSSION TITLE
Academic conference 2024 orientation, 2023-12-29 Fri 23:00

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#337] [General]
안녕하세요.

Academic conference 2024 오리엔테이션이  12월 29일 금요일 오후 11시에 진행됩니다.
올해는 제가 google one 계정을 프리미엄 요금제로 결제했기 때문에
시간 제한이 없고 품질이 뛰어난 google meet로 진행합니다.

Academic conference 2024, orientation
29 December 2023, 11:00pm – 30 December 2023, 12:00am · 
Time zone: Asia/Seoul
Google Meet joining info
Video call link: https://meet.google.com/wri-aaif-hhc

진행 순서는 다음과 같습니다.

- 각자 자기 소개
- Academic conference 2024 진행 방식 소개
- 책 읽는 순서 결정
- 워크샵 여행지 소개
입니다.

내년에도 재밌고 유익하게 진행했으면 좋겠습니다.

**Comment by yeslee-v** [comment]
아, 제가 저 때 참여를 못했었는데 워크샵 여행지는 어디로 결정되었나요?

**Reply by jongfeel** [reply]
결정은 작년 워크샵 때 강릉으로 결정 했고
오리엔테이션 날에는 리마인드만 했습니다.

워크샵 여행지로 좋은 곳 소개해 주시면 변경도 됩니다.
